packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -36,10 +39,10 @@ jest.mock('uuid', () => {
   };
 });
 
-const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
+const stubCreateEthers = (ctrl: TokensController, res: () => boolean) => {
   return sinon.stub(ctrl, '_createEthersContract').callsFake(() => {
     return {
-      supportsInterface: sinon.stub().returns(res),
+      supportsInterface: sinon.stub().returns(res()),
```

Comment:
?
```
const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
  return sinon.stub(ctrl, '_createEthersContract').callsFake(() => {
    return {
      supportsInterface: sinon.stub().resolves(res),
```

Reply:
Either `returns` or `resolves` seems to work.  But I think `res` being a function is useful. The stub can be setup once in `beforeEach` with a default value, while allowing individual tests can overwrite the default.  Otherwise every test would need to setup the stub.
 
https://github.com/MetaMask/core/blob/7d22741b4875bd28dc045cadf7b2fe4317358c20/packages/assets-controllers/src/TokensController.test.ts#L1148